contracts/core/interfaces/IRebalancingSetToken.sol

```diff
@@ -23,13 +23,27 @@ pragma solidity 0.4.24;
  * The IRebalancingSetToken interface provides a light-weight, structured way to interact with the
  * RebalancingSetToken contract from another contract.
  */
+
 interface IRebalancingSetToken {
+    /*
+     * Place bid during rebalance auction. Can only be called by Core.
+     *
+     * @param _quantity                 The amount of currentSet to be rebalanced
+     * @return combinedTokenArray       Array of token addresses invovled in rebalancing
```

Comment:
The return type is usually denoted here, more effective than the variable name. So this would be

@return address[]
@return uint256[]
@return uint256[]